### ISSUE TITLE
Discussion: Locations of various levels of documentation

### ISSUE THREAD

**Issue by mradamcox** [archesproject/arches#1223]
We've talked about how we need different levels of documentation, to accommodate different types of users. We have a good deal of content, either planned, in progress, or spread between existing sources. I've been tracking this content [here](https://docs.google.com/spreadsheets/d/1zR5xGMg7mLY9NpWGlv9F8_QaGeqgZDp7GGoG0U-l1YQ/edit?usp=sharing) (which is pretty much just a more detailed version of the [index](https://docs.google.com/document/d/1IkrcaqNPtV6dA210GT-Nbj5hWVN0lzSwE_QK7fWyWYc/edit?usp=sharing) I already shared).

At this point, I'm thinking that all developer-specific documentation will be here on github in the wiki. We will have full set of "user-oriented" documentation hosted in the main docs (either readthedocs or docbox) to hold the general info, description of how to configure arches, manage resources, etc. Finally, there will be some in-app documentation as described in #1181, #1182, and #1183.

Here's more detail description of what would go where.

- Developer Documentation: all in wiki
    - dev installation directions
    - system architecture diagram
    - all data model documentation
    - API documentation
    - page to document all of the CLI operations
    - basic and advanced production configuration? or just advanced?
    - developer-specific glossary?

- General User Documentation: in main docs
    - welcome, intro to arches, how to contribute (pointing to github, etc.)
    - system requirements (cpu, memory)
    - dependencies
    - quick installation (download links for wizard), with link to wiki for dev installation
    - Configuring Arches (this is a catchall term for "initial customization" topics, like setting up the map server, "dashboard", system settings, etc.
    - field project manager documentation
    - import/export user interface
    - should basic production configuration documentation go here instead of the wiki?

- Resource Manager User Documentation: both in-app and in main docs
    - step guides/walkthroughs for creating resource models and branches
    - full documentation of the various arches designer tools
    - in-app would be the panel described in #1181, which will link back to main docs in some cases

- Resource Creator: both in-app and in main docs
    - general introduction to resource creation and editing
    - more detailed information on how to use certain widgets, like map widget, etc.
    - in-app would be the panel described in #1182, and would point back to main docs.

Discussion questions:
Does this setup raises red flags? or if there's another aspect of documentation I have left out?
Should we expect more in-app documentation to be necessary, once more modules, like the field project manager, system settings, map server, etc. are completed?

**Comment by mradamcox** [comment]
Update on all of this: I've decided to go back to RTD for the user manual, as I never found good enough reasons for the switch to docbox. #1495 covers the bulk of this transition.

I also talked with @annabelleee last week a little bit about this, and we discussed the benefit of trying to place as much of the documentation into one place. Specifically, moving more of the dev documentation into RTD from the github wiki. After #1495 is complete, it will be easier to evaluate this idea. My main concern is that there will just be too much content in RTD.
